### ISSUE TITLE
Problem z zapasowym językiem napisów

### ISSUE THREAD

**Issue by Rumcaajs** [QNapi/qnapi#122]
Witam.
Korzystam z qnapi od dłuższego czasu i bardzo sobie chwalę ten soft. Do tej pory wszystko działało poprawnie. W chwili obecnej mam problem z zapasowym językiem napisów. Napisy pobieram w języku polskim (primary) i zawsze ustawiałem zapasowy jako "none". Po zapisaniu to działało poprawnie. W chwili obecnej po zmianie tego pola na "none" i wejściu ponownie w ustawienia dalej jest widoczny zapasowy język jako angielski. Robiłem tak kilkukrotnie, bez zmian. Tak samo z reinstalacją oprogramowania, dalej jeśli nie ma napisów w języku polskim to są pobierane w języku angielskim.
![qnapi](https://user-images.githubusercontent.com/29737706/27603407-4a727e28-5b76-11e7-91a8-fae89db0d57a.png)

**Comment by Rumcaajs** [comment]
OS Win10 x64

**Comment by uded** [comment]
U mnie jest na odwrót - nie mogę za nic ustawić zapasowego języka napisów! Pole wraca do wartości pustej. Dla odmiany na MacOS X.

**Comment by C4rl0s79** [comment]
Angielski jest wyświetlany i pobierany jako drugi, niezależnie od wpisanej opcji w pliku ini, nawet po ręcznej edycji i ustawieniu backup_language=pl to jako drugi język jest ustawiony angielski i pobierane są angielskie napisy

**Comment by koRnac** [comment]
nie pomaga nawet zmiana pliku ini na "tylko do odczytu" ze zmienionymi wcześniej swoimi ustawieniami.

**Comment by useybrain** [comment]
Ten sam problem!

**Comment by xtoudi** [comment]
Potwierdzam, zapasowy język nie można zmienić - zawsze jest Angielski.
MacOS High Sierra oraz MacOS Mojave

**Comment by Adrianwsmski** [comment]
Potwierdzam
Czy też macie problem związanym z plikiem z qnapi który posiada napisy w języku polskim ale
vlc mimo wszystko odtwarza je w języku angielskim ?? Help

**Comment by useybrain** [comment]
@Adrianwsmski tak. Usuń wszsytkie inne napisy. Zmieć nazwę napisów na takie same jak film bez końcowki .pl bo tak ostatnio mi zaczął dodawać qnapi!

**Comment by Adrianwsmski** [comment]
@useybrain usunąłem napisy zewsząd skopiowałem nazwę filmu do napisów a mimo wszystko dodaje napisy po angielsku :/

**Comment by cadaver123** [comment]
Zapasowy jezyk odczytywany jest z ustawień z pola "backup_language", natomiast zapisywany do pola "language_backup" - stąd problemy. Stąd, żeby go ustawić, trzeba w pliku z ustawieniami (na windowsie C:\Users\<USER>\AppData\Roaming\qnapi.ini) w sekcji [qnapi] dodać ręcznie wpis backup_language z wybranym przez nas językiem, albo samo "backup_language=", jęśli nie chcemy korzystać z tej opcji.

**Comment by tpluscode** [comment]
@cadaver123 wiesz może gdzie ten plik znaleźć na Mac'u?

**Comment by JMarciniak** [comment]
@tpluscode plik jest w ~/.config/qnapi.ini
musisz mieć 2 wpisy:
backup_language=en
language_backup=en

zamiast en wstaw język jaki chcesz wstawić.

**Comment by bentkowski** [comment]
Identyczny problem na ubuntu 18.04, ręczna edycja pliku ~/.config/qnapi.ini pomaga
